src/components/ArticleCard.jsx

```diff
@@ -1,21 +1,20 @@
 import "../App.css"
+import { Link } from "react-router-dom";
+
 function ArticleCard({ article }) {
-    
+
     return (
-        <>
             <article className="card_container">
```

Comment:
I'd wrap your link around the whole card to make the clickable area more accessable